### PR TITLE
Build also python27 wheel for libsonata

### PR DESCRIPTION
**Pull request by wizmer**

### CODE REVIEW

**Comment by pramodk** [comment]
@wizmer : Is providing Python 2.7 wheel considered as promoting Python 2? :grin:

**Comment by wizmer** [comment]
LOL. Unfortunately, as long as bluepy supports python 2.7, I have no other choice than to build these wheels...
But don't worry, it seems that the fact that nexus-forge is python 3 only was the providential argument we needed to change the mentalities and bluepy should drop python 2 support in the following weeks.
:crossed_fingers:

**Comment by pramodk** [comment]
>  But don't worry, it seems that the fact that nexus-forge is python 3 only was the providential argument we needed to change the mentalities and bluepy should drop python 2 support in the following weeks.

Good to hear! We can remember this covid-19 time for getting rid of Python2 from bluepy! :grin:

**Review by matz-e — COMMENTED** [review]
But why if we're just going to drop it in a few weeks?

**Comment by wizmer** [comment]
Bluepy currently needs a bug fix because of h5py>=3 that I can't wait a few weeks before merging.

File: .github/workflows/publish-sdist-wheels.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ env:
   MACOSX_DEPLOYMENT_TARGET: '10.13'
   CIBW_BUILD_VERBOSITY: 2
   # Only build on Python 3.x
-  CIBW_BUILD: 'cp3?-*'
+  CIBW_BUILD: 'cp3?-* cp27-*'
```
Comment (tomdele) [inline .github/workflows/publish-sdist-wheels.yml:13]: I tried before and this fails due to cibuildwheel version compatibility problems when running with 1.6.4. 
You should relax the version number to make it works probably. But this is a bit scary to have wheels made with different versions ....

Comment (wizmer) [reply .github/workflows/publish-sdist-wheels.yml:13]: Ok.....
Let's drop python 2 instead...

**Comment by wizmer** [comment]
Bad idea

**Pull request closed without merging**